static/src/javascripts/projects/membership/payment-form.js

```diff
@@ -50,44 +44,53 @@ define([
             TIER_FIELD: 'js-tier-field',
             UPDATING: 'js-updating'
         },
-        DOM: {},
         data: {
             CARD_TYPE: 'data-card-type'
         }
     };
 
+    function StripePaymentForm(form, successElement, url) {
```

Comment:
Ah yes that is rather inconsistent of me, I might change both `form` and `successElement` to be CSS selectors rather than any kind of DOM element, then the constructor can wrap them with bonzo